text/src/autogluon/text/automm/data/utils.py

```diff
@@ -0,0 +1,17 @@
+from typing import Union, Optional, List, Tuple
+
+
+def get_default_config_value(
+        config: dict,
+        keys: Tuple[str, ...],
+):
```

Comment:
The function name does not match what it is doing. This looks to me more like filter to only allowed values.

Reply:
Name changed to `extract_value_from_config` because we try to extract some default hyper-parameter's value from a config.